packages/web/src/views/launchpad/containers/launchpad-main-container/LaunchpadMainContainer.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react";
+import Link from "next/link";
+
+import { ThemeKeys } from "@styles/ThemeTypes";
+import Button from "@components/common/button/Button";
+import { ButtonStyleProps } from "@components/common/button/Button.styles";
+import IconRightArrow from "@components/common/icons/IconRightArrow";
+import { Divider } from "@components/common/divider/divider";
+
+import { useGetLaunchpadSummary } from "@query/launchpad/use-get-launchpad-summary";
+
+interface LaunchpadMainContainerProps {
+  themeKey: ThemeKeys;
+  icon: React.ReactNode;
+}
+
+const LaunchpadMainContainer: React.FC<LaunchpadMainContainerProps> = ({
+  themeKey,
+  icon,
+}) => {
+  const { data: launchpadSummary } = useGetLaunchpadSummary();
+
+  const defaultStyle: ButtonStyleProps = {
+    textColor: "text32",
+    bgColor: themeKey === "dark" ? "border02" : "background04",
+  };
+
+  return (
+    <>
+      <section className="launchpad-section">
+        <div className="launchpad-title-container">
+          <h3 className="title">GnoSwap Launchpad&nbsp;</h3>
+          <h4 className="sub-title">
+            Where Pioneers and Supporters Unite — Discover <br />
+            the Next Big Thing on gno.land, Powered by $GNS.
+          </h4>
+        </div>
+        <div className="launchpad-button-wrapper">
+          <Link href="https://docs.gnoswap.io" target="_blank">
```

Comment:
Please use static variables for the following paths.
`packages/web/src/constants/external-url.contant.ts`